tests/unit_tests/test_blog.py

```
@@ -114,6 +135,28 @@ def test_tag_filter(test_app):
     assert not post_contents_on_page(response)
 
 
+def test_post_view_generic_exception(test_app):
+    """Test that post view handles generic exceptions correctly."""
+    # Set up the mock to raise an exception
+    test_app.application.db.get_post.side_effect = Exception("Missing required field: 'title'")
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)